gui/wxpython/photo2image/ip2i_manager.py

```diff
@@ -453,37 +448,37 @@ def __init__(
 
         self.GCPcount = 0
         try:
-            f = open(self.file["points"], mode="w")
-            # use os.linesep or '\n' here ???
-            f.write("# Ground Control Points File\n")
-            f.write("# \n")
-            f.write("# target location: " + self.currentlocation + "\n")
-            f.write("# target mapset: " + self.currentmapset + "\n")
-            f.write("#\tsource\t\ttarget\t\tstatus\n")
-            f.write("#\teast\tnorth\teast\tnorth\t(1=ok, 0=ignore)\n")
-            f.write(
-                "#-----------------------     -----------------------     ---------------\n"  # noqa: E501
-            )
-
-            check = "0"
-            for index in range(numberOfFiducial):
-                coordX0 = "0"
-                coordY0 = "0"
-                coordX1 = dataFiducialX[index]
-                coordY1 = dataFiducialY[index]
+            with open(self.file["points"], mode="w") as f:
+                # use os.linesep or '\n' here ???
+                f.write("# Ground Control Points File\n")
+                f.write("# \n")
+                f.write("# target location: " + self.currentlocation + "\n")
+                f.write("# target mapset: " + self.currentmapset + "\n")
+                f.write("#\tsource\t\ttarget\t\tstatus\n")
+                f.write("#\teast\tnorth\teast\tnorth\t(1=ok, 0=ignore)\n")
                 f.write(
-                    coordX0
-                    + " "
-                    + coordY0
-                    + "     "
-                    + coordX1
-                    + " "
-                    + coordY1
-                    + "     "
-                    + check
-                    + "\n"
+                    "#-----------------------     -----------------------     ---------------\n"  # noqa: E501
                 )
 
+                check = "0"
+                for index in range(numberOfFiducial):
+                    coordX0 = "0"
+                    coordY0 = "0"
+                    coordX1 = dataFiducialX[index]
+                    coordY1 = dataFiducialY[index]
+                    f.write(
+                        coordX0
+                        + " "
+                        + coordY0
+                        + "     "
+                        + coordX1
+                        + " "
+                        + coordY1
+                        + "     "
+                        + check
+                        + "\n"
+                    )
+
```

Comment:
The f.close() at line 495 below might be invalid outside the context manager. Reviewing your other PR made me think of it.